apps/src/templates/levelSummary/CheckForUnderstanding.jsx

```diff
@@ -23,6 +23,9 @@ const CheckForUnderstanding = ({
 }) => {
   const currentLevel = levels.find(l => l.activeId === currentLevelId);
   const nextLevel = levels.find(l => l.position === currentLevel.position + 1);
+  const sectionParam = selectedSection?.id
```

Comment:
whoa, i didn't realize we had access to this `?.` operator yet!

Reply:
I think it's just in the apps directory (https://codedotorg.slack.com/archives/C0T0PNR0D/p1680630902632839?thread_ts=1680562949.098389&cid=C0T0PNR0D)